apps/src/blockly/addons/cdoUtils.ts

```diff
@@ -795,7 +795,7 @@ export function getAllGeneratedCode(extraCode?: string) {
       if (workspace) {
         Blockly.getGenerator().init(workspace);
         const blocks = workspace.getTopBlocks(true);
-        const blocksCode: GoogleBlockly.Block[] = [];
+        const blocksCode: (string | [string, number])[] = [];
```

Comment:
I'm unsure why this was previously designated as a block array when it's actually the student's code.

Reply:
probably my bad 🤷‍♀️